### DISCUSSION TITLE
Homarr 0.15.7 fail upfate

### DISCUSSION THREAD

**Discussion by rezterra** [community-scripts/ProxmoxVE#139] [Q&A] [answered]
Hi, when trying to update homarr I get the following error:
- Updating Homarr to 0.15.7   
[ERROR] in line 80: exit code 0: while executing command yarn build &> /dev/null

After this the service has stopped working.

**Comment by MickLesk** [comment]
Storage full? RAM Overflow? 

At my system it runs fine.

**Comment by kiwinh0** [comment]
The same thing happens to me. When you run "update" directly it tells you that there are no updates and if you paste the installation command in the hommar terminal it breaks directly and puts the information that says @rezterra

**Comment by havardthom** [comment, marked as answer]
This will be fixed in https://github.com/community-scripts/ProxmoxVE/pull/162

There was a bug in the homarr update function and also some DB backup was missing. Unfortunately that means that users who got to the `yarn build` error has lost their homarr DB (storing mostly users/settings i think). Dashboard configs should still be intact (these are not stored in DB, but in separate config files).

**Comment by rezterra** [comment]
Finally create a new lxc and restore /opt/homarr/data

**Reply by havardthom** [reply]
`/opt/homarr/data` only contains configs for dashboards, if you used helper script to install Homarr LXC you should also restore the database which is stored in `/opt/homarr/database/db.sqlite`
